tests/02_eradiate/01_unit/experiments/test_dem.py

```diff
@@ -0,0 +1,282 @@
+import mitsuba as mi
+import numpy as np
+import pytest
+
+import eradiate
+from eradiate import unit_registry as ureg
+from eradiate.contexts import KernelDictContext
+from eradiate.experiments._dem import DEMExperiment
```

Comment:
You also want to check that your lazy imports work:
```suggestion
from eradiate.experiments import DEMExperiment
```